Common/GPU/Vulkan/VulkanQueueRunner.cpp

```diff
@@ -1336,7 +1338,8 @@ void VulkanQueueRunner::PerformRenderPass(const VKRStep &step, VkCommandBuffer c
 
 		case VKRRenderCommand::DRAW_INDEXED:
 			if (pipelineOK) {
-				vkCmdBindDescriptorSets(cmd, VK_PIPELINE_BIND_POINT_GRAPHICS, pipelineLayout, 0, 1, &c.drawIndexed.ds, c.drawIndexed.numUboOffsets, c.drawIndexed.uboOffsets);
+				VkDescriptorSet set = (*descSets)[c.drawIndexed.descSetIndex].set;
```

Comment:
Maybe deserves a `_dbg_assert_(set != VK_NULL_HANDLE);` here as well, just for symmetry and to ensure no false sense of security when skimming the code?

-[Unknown]

Reply:
Yeah.